sdk/ml/azure-ai-ml/azure/ai/ml/entities/_workspace_hub/workspace_hub_config.py

```diff
@@ -20,6 +20,7 @@
 class WorkspaceHubConfig:
     def __init__(
         self,
+        *,
         additional_workspace_storage_accounts: Optional[List[str]] = None,
         default_workspace_resource_group: Optional[str] = None,
     ) -> None:
```

Comment:
The docstring after this should be

```
        :keyword additional_workspace_storage_accounts: A list of resource IDs of existing storage accounts that will be
            utilized in addition to the default one.
        :paramtype additional_workspace_storage_accounts: List[str]
        :keyword default_workspace_resource_group: A destination resource group for any Project workspaces that join the
            workspaceHub, it will be the workspaceHub's resource group by default.
        :paramtype default_workspace_resource_group: str
```